src/ansible_navigator/actions/inventory.py

```diff
@@ -19,8 +18,10 @@
 from ..runner import AnsibleInventory
 from ..runner import Command
 from ..steps import Step
+from ..ui_framework import Color
```

Comment:
(nit) it'd be clearer if this was
```suggestion
from ..ui_framework.ui_constants import Color
```
Well, even
```suggestion
from ..ui_framework.constants import Color
```